erpnext/quality_management/doctype/quality_review/quality_review.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+from frappe.model.document import Document
+
+class QualityReview(Document):
+	def validate(self):
+		if self.measurable == "Yes":
```

Comment:
There's a lot of code duplication here. Only make this check where it differs from the `else` statement.

Reply:
Resolved